packages/nimble-components/src/table-column/menu-button/cell-view/styles.ts

```diff
@@ -0,0 +1,23 @@
+import { css } from '@microsoft/fast-element';
```

Comment:
What is the state of the visual design right now? Assuming it's not final, do we have an item on Brandon's backlog to provide one? If we think it might be a little while then I guess we could put the docs in the Incubating section initially but I don't feel strongly that's necessary.

Personally I find the bold text seems to overemphasize the column relative to the others.

Reply:
Great question. Last I heard, the visual design was in work. However, that was a few weeks ago. I need to follow up with Fred and Brandon.

Reply:
Since it isn't straightforward to make these styling changes to the button, I'll create an issue to tackle this as a follow up task. This way, we can have the necessary discussions around the best way to achieve the desired styling and double check that everything looks as expected with focus/hover states while not blocking the next stages of the column work.

Reply:
Created #2247